Comp2025/src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -101,21 +104,19 @@ private enum WristMode
   //      Comp     -177.3  -176.3     -124.7    24.9      25.8
   //      Practice -177.8  -176.8     -124.7    27.3      27.4
   private static final double       kWristAngleRetracted      = Robot.isComp( ) ? -176.3 : -176.8;  // One degree from hardstops
-  private static final double       kWristAngleDeployed       = Robot.isComp( ) ? 24.9 : 27.3;      //
+  // private static final double       kWristAngleDeployed       = Robot.isComp( ) ? 24.9 : 27.3;    Currently being kept for reference
 
-  private static final double       kWristAngleStowed         = 0;
   private static final double       kWristAngleCoralL1        = 0;
   private static final double       kWristAngleCoralL23       = 0;
   private static final double       kWristAngleCoralL4        = 0;
   private static final double       kWristAngleCoralStation   = 0;
 
-  private static final double       kWristAngleAlgaeL23       = 0;
-  private static final double       kWristAngleAlgaeL34       = 0;
+  private static final double       kWristAngleAlgaeReef      = 0;
   private static final double       kWristAngleAlgaeProcessor = 0;
   private static final double       kWristAngleAlgaeNet       = 0;
 
   private static final double       kWristAngleMin            = kWristAngleRetracted;
-  private static final double       kWristAngleMax            = kWristAngleDeployed;
+  private static final double       kWristAngleMax            = kWristAngleAlgaeReef;
```

Comment:
Max is not necessarily the same angle as acquiring algae from the reef. It's really how far we will let the mechanism travel under any circumstances. This is probably further than any position used during the match, because we don't want to command movements right into the end of travel.